src/cli.rs

```diff
@@ -30,35 +32,51 @@ impl Default for Options {
             ref_test: false,
             columns: Column(80),
             lines: Line(24),
-            title: "Alacritty".to_owned()
+            title: DEFAULT_TITLE.to_owned()
         }
     }
 }
 
 impl Options {
-    /// Iterate through env::args() to build `Options`
+    /// Build `Options` from command line arguments
     pub fn load() -> Options {
         let mut options = Options::default();
-        let mut args_iter = env::args();
 
-        while let Some(arg) = args_iter.next() {
-            match &arg[..] {
-                // Generate ref test
-                "--ref-test" => options.ref_test = true,
-                "--print-events" => options.print_events = true,
-                // Set dimensions
-                "-d" | "--dimensions" => {
-                    args_iter.next()
-                        .map(|w| w.parse().map(|w| options.columns = Column(w)));
-                    args_iter.next()
-                        .map(|h| h.parse().map(|h| options.lines = Line(h)));
-                },
-                "-t" | "--title" => {
-                    args_iter.next().map(|t| options.title = t);
-                },
-                // ignore unexpected
-                _ => (),
-            }
+        let matches = App::new("alacritty")
+            .version(crate_version!())
+            .about("GPU-accelerated terminal emulator")
```

Comment:
imo we should add the description to the Cargo.toml and use `crate_description!()` here